sdk/tests/conformance2/vertex_arrays/vertex-array-object.html

```diff
@@ -476,17 +476,22 @@
     // delete the color buffers AND the position buffer.
     gl.bindVertexArray(null);
     for (var ii = 0; ii < colorBuffers.length; ++ii) {
-      gl.deleteBuffer(colorBuffers[ii]);
-      gl.deleteBuffer(elementBuffers[ii]);
       gl.bindVertexArray(vaos[ii]);
-      var boundBuffer = gl.getVertexAttrib(1, gl.VERTEX_ATTRIB_ARRAY_BUFFER_BINDING);
+
+      gl.deleteBuffer(colorBuffers[ii]);
+      gl.deleteBuffer(positionBuffer);
```

Comment:
If the buffer is automatic unbound, it should be not "in use" status. 
You can first deleteBuffer then bindVertexArray. You can see the deleted buffer can still be used.